package.json

```diff
@@ -49,7 +49,7 @@
     "test": "npm run test:main && npm run test:fp",
     "test:fp": "node test/test-fp",
     "test:main": "node test/test",
-    "validate": "npm run style & npm run test"
+    "validate": "npm run style & npm run test & wait"
   },
```

Comment:
The issue seems to be that `ghooks` does not find its own `opt-cli` dependency when running its own `install` script on `npm@"^2.0.0."`.

[ghooks issue #71](https://github.com/gtramontina/ghooks/issues/71)